modules/imap/handler_modules.php

```diff
@@ -1541,8 +1542,9 @@ public function process() {
                 }
             }
             if (array_key_exists('default', $server) && $server['default']) {
-                $default_server_id = $id;
+                $has_default = true;
             }
+            $default_server_id = $id;
```

Comment:
Why default_server_Id is every ID in the imap list? Shouldn't it be only the one marked as default or maybe the first one?